src/__tests__/managers/ContractManager.test.ts

```diff
@@ -1892,3 +1892,48 @@ describe('ContractManager changeTransaction tests', () => {
     ).rejects.toThrow(errors.NOT_BIGNUMBERISH('value'))
   })
 })
+
+describe('Utils of ContractManager', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+    jest.restoreAllMocks()
+  })
+
+  it('Should return true if ContractManager abi implements the function', () => {
+    const functionName = 'allowance'
+    const params = ['address', 'address']
+    const signature = `${functionName}(${params.join(',')})`
+    const sdk = new ContractManager()
+    const abi = new ethers.utils.Interface([
+      {
+        inputs: [
+          {
+            name: 'owner',
+            type: 'address'
+          },
+          {
+            name: 'spender',
+            type: 'address'
+          }
+        ],
+        name: 'allowance',
+        outputs: [],
+        stateMutability: 'view',
+        type: 'function'
+      }
+    ])
+    sdk['_abi'] = abi
+    const spyGetFuntion = jest.spyOn(abi, 'getFunction')
```

Comment:
You should mock the call to the `implementsFunction` method instead of the getFunction, otherwise you are testing the implementsFunction in the test of the ContractManager implements function in a unit test.